### PR TITLE
Added recipe for extension-helpers

### PR DESCRIPTION
**Pull request by astrofrog**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

@mwcraig @bsipocz - can you confirm you are happy to be co-maintainers?

### CODE REVIEW

File: recipes/extension-helpers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  imports:
+    - extension_helpers
+
+about:
+  home: https://github.com/astropy/extension-helpers
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Utilities for building and installing packages with compiled extensions'
+
```
Comment (mwcraig) [inline recipes/extension-helpers/meta.yaml]: Minor, but can you please remove the blank line?

**Comment by mwcraig** [comment]
I am happy to be a maintainer!

**Comment by bsipocz** [comment]
> can you confirm you are happy to be co-maintainers?

of course!

**Comment by astrofrog** [comment]
@conda-forge/help-python - this is ready for review

**Comment by mwcraig** [comment]
Looks good, merging!

**Pull request merged**